opsgenie/resource_opsgenie_team.go

```diff
@@ -3,10 +3,11 @@ package opsgenie
 import (
 	"context"
 	"errors"
+	"log"
+
 	"github.com/opsgenie/opsgenie-go-sdk-v2/client"
 	"github.com/opsgenie/opsgenie-go-sdk-v2/escalation"
 	"github.com/opsgenie/opsgenie-go-sdk-v2/schedule"
-	"log"
```

Comment:
the _import_ looks like `gci` could be of help. github.com/daixiang0/gci/

Reply:
Yeah, let me drop it, this change is a bit irrelevant for this fix